mkt/feed/tests/test_commands.py

```diff
@@ -0,0 +1,41 @@
+from django.core.management import call_command
+
+from mpconstants import collection_colors as colors
+from nose.tools import eq_, ok_
+
+from mkt.feed.models import FeedApp, FeedCollection
+from mkt.site.tests import app_factory, TestCase
+
+
+class TestMigrateCollectionColor(TestCase):
+
+    def setUp(self):
+        self.color_hex = colors.COLLECTION_COLORS_REVERSE.keys()[0]
+        self.color_name = colors.COLLECTION_COLORS_REVERSE[self.color_hex]
+
+    def test_app(self):
+        obj = FeedApp.objects.create(app=app_factory(),
+                                     background_color=self.color_hex)
+        eq_(obj.color, None)
+        call_command('migrate_collection_colors')
```

Comment:
Since the command actually kicks off an async task to change the colors, I think there is a race condition in these tests. It might make more sense to test the task itself, mock the task here, and make sure that `mocked_task.delay.call_count` is 1.